security/pfSense-pkg-acme/files/usr/local/pkg/acme/acme.sh

```diff
@@ -0,0 +1,4587 @@
+#!/usr/bin/env sh
```

Comment:
I didn't notice that, but yeah it should be sufficient in both cases. I just noticed when looking at the diff that it was 644.